test_bot/__main__.py

```diff
@@ -98,6 +111,7 @@ async def on_message_command_error(
 
 print(f"disnake: {disnake.__version__}\n")
 
-bot = TestBot()
-bot.load_all_extensions("cogs")
-bot.run(os.environ.get("BOT_TOKEN"))
+if __name__ == "__main__":
+    bot = TestBot()
+    bot.load_all_extensions(Config.cogs_folder)
```

Comment:
```suggestion
    bot.load_extensions(os.path.join("test_bot", Config.cogs_folder))
```
Removes the need for `load_all_extensions` above, now that `load_extensions` exists

Reply:
https://github.com/DisnakeDev/disnake/pull/247/commits/daa8d44a721ec60a4b7347f1a9fb1de145ee1f56